### PR TITLE
chore: track the terraform version in README.adoc with updatecli

### PR DESCRIPTION
**Pull request by smerle33**
as per https://github.com/jenkins-infra/shared-tools/issues/11

track terraform version and update README.adoc

### CODE REVIEW

File: updatecli/updatecli.d/hashicorp-tools.yml [hunk 1]
```diff
@@ -20,9 +20,21 @@ sources:
       repository: "docker-hashicorp-tools"
       token: "{{ requiredEnv .github.token }}"
       username: "{{ .github.username }}"
+  terraformHashicorpToolsVersion:
+    depends_on:
+      - dockerHashicorpToolsImageVersion
+    kind: file
+    spec:
+      file: "https://raw.githubusercontent.com/jenkins-infra/docker-hashicorp-tools/0.4.4/Dockerfile"
```
Comment (lemeurherve) [inline updatecli/updatecli.d/hashicorp-tools.yml:28]:
```suggestion
      file: "https://raw.githubusercontent.com/jenkins-infra/docker-hashicorp-tools/{{ source `dockerHashicorpToolsImageVersion` }}/Dockerfile"
```

File: updatecli/updatecli.d/hashicorp-tools.yml [hunk 2]
```diff
@@ -41,13 +54,23 @@ targets:
...
   default:
     kind: github
     scmID: default
     targets:
       - updateJenkinsFile
+      - updateReadme
     spec:
       labels:
         - dependencies
+        - HashicorpTools
```
Comment (lemeurherve) [inline updatecli/updatecli.d/hashicorp-tools.yml:76]:
```suggestion
        - hashicorp-tools
```

**Review by lemeurherve — CHANGES_REQUESTED** [review]

File: updatecli/updatecli.d/hashicorp-tools.yml (hunk 1 above)
Comment (smerle33) [reply updatecli/updatecli.d/hashicorp-tools.yml]: great catch !! tks ...

**Review by dduportal — APPROVED** [review]
LGTM great job!

```
[](https://infra.ci.jenkins.io/blue/organizations/jenkins/shared-tools/detail/PR-12/4/pipeline/20#step-24-log-130)[](https://infra.ci.jenkins.io/blue/organizations/jenkins/shared-tools/detail/PR-12/4/pipeline/20#step-24-log-131)[](https://infra.ci.jenkins.io/blue/organizations/jenkins/shared-tools/detail/PR-12/4/pipeline/20#step-24-log-132)[](https://infra.ci.jenkins.io/blue/organizations/jenkins/shared-tools/detail/PR-12/4/pipeline/20#step-24-log-133)[](https://infra.ci.jenkins.io/blue/organizations/jenkins/shared-tools/detail/PR-12/4/pipeline/20#step-24-log-134)[](https://infra.ci.jenkins.io/blue/organizations/jenkins/shared-tools/detail/PR-12/4/pipeline/20#step-24-log-135)[](https://infra.ci.jenkins.io/blue/organizations/jenkins/shared-tools/detail/PR-12/4/pipeline/20#step-24-log-136)[](https://infra.ci.jenkins.io/blue/organizations/jenkins/shared-tools/detail/PR-12/4/pipeline/20#step-24-log-137)[](https://infra.ci.jenkins.io/blue/organizations/jenkins/shared-tools/detail/PR-12/4/pipeline/20#step-24-log-138)[](https://infra.ci.jenkins.io/blue/organizations/jenkins/shared-tools/detail/PR-12/4/pipeline/20#step-24-log-139)[](https://infra.ci.jenkins.io/blue/organizations/jenkins/shared-tools/detail/PR-12/4/pipeline/20#step-24-log-140)[](https://infra.ci.jenkins.io/blue/organizations/jenkins/shared-tools/detail/PR-12/4/pipeline/20#step-24-log-141)[](https://infra.ci.jenkins.io/blue/organizations/jenkins/shared-tools/detail/PR-12/4/pipeline/20#step-24-log-142)[](https://infra.ci.jenkins.io/blue/organizations/jenkins/shared-tools/detail/PR-12/4/pipeline/20#step-24-log-143)[](https://infra.ci.jenkins.io/blue/organizations/jenkins/shared-tools/detail/PR-12/4/pipeline/20#step-24-log-144)TARGETS

========


updateReadme

------------


**Dry Run enabled**


⚠ File content for "./terraform/README.adoc", updated. 


--- ./terraform/README.adoc

+++ ./terraform/README.adoc

@@ -5,7 +5,7 @@

 == Requirements

 

 * The following tools (you may check the Docker image used by the CI at https://github.com/jenkins-infra/docker-hashicorp-tools[](https://infra.ci.jenkins.io/blue/organizations/jenkins/shared-tools/detail/PR-12/4/pipeline/20#step-24-log-145)[](https://infra.ci.jenkins.io/blue/organizations/jenkins/shared-tools/detail/PR-12/4/pipeline/20#step-24-log-146)[](https://infra.ci.jenkins.io/blue/organizations/jenkins/shared-tools/detail/PR-12/4/pipeline/20#step-24-log-147)[](https://infra.ci.jenkins.io/blue/organizations/jenkins/shared-tools/detail/PR-12/4/pipeline/20#step-24-log-148) with the up-to-date requirements):

-** Terraform version 1.0.11 CLI to provision the infrastructure

+** Terraform version 1.1.7 CLI to provision the infrastructure

 ** GNU `make` to ease running the tasks involved in this provisioning

 ** link:https://github.com/tfsec/tfsec[tfsec][](https://infra.ci.jenkins.io/blue/organizations/jenkins/shared-tools/detail/PR-12/4/pipeline/20#step-24-log-149)

 ** link:https://golang.org/[Golang] to allow testing with link:https://terratest.gruntwork.io/[terratest]
 ```

**Pull request merged**